AI/modules/signal/models/PatchTST/train.py

```
@@ -1,103 +1,315 @@
+
+    # 종목별로 피처 계산
+    print(">> 피처 계산 중 (일봉 + 주봉/월봉)...")
+    processed = []
+    for ticker in tqdm(raw_df['ticker'].unique(), desc="피처 계산"):
+        df_t = raw_df[raw_df['ticker'] == ticker].copy()
+        try:
+            df_t = add_technical_indicators(df_t)       # 일봉 11개
+            df_t = add_multi_timeframe_features(df_t)   # 주봉/월봉 6개
+            processed.append(df_t)
+        except Exception:
+            continue
```

Ruff (0.15.6)

[error] `try`-`except`-`continue` detected, consider logging the exception

(S112)